xbmc/cores/VideoPlayer/VideoPlayerAudioID3.cpp

```diff
@@ -251,28 +265,83 @@ void CVideoPlayerAudioID3::ProcessID3v2(const ID3v2::Tag* tag) const
     MUSIC_INFO::CMusicInfoTag* currentMusic = currentMusicFile.GetMusicInfoTag();
     if (currentMusic)
     {
+      if (m_currentRadiotext)
+        m_currentRadiotext->SetPlayingRadioText(true);
+
       bool changed = false;
 
       const ID3v2::FrameListMap& frameListMap = tag->frameListMap();
       for (const auto& it : frameListMap)
       {
-        if (it.first == "TIT2")
+        if (it.first[0] == 'T')
         {
-          const std::string str = it.second.front()->toString().to8Bit(true);
-          currentMusic->SetTitle(str);
-          changed = true;
-        }
+          std::string str = it.second.front()->toString().to8Bit(true);
+          if (str == "!!!EMPTY!!!")
+            str = "";
```

Comment:
Whatever we do (patch taglib to not ignore tags with empty value?), this hack must go away!